src/inference/dev_api/threading/ie_cpu_streams_executor.hpp

```diff
@@ -47,10 +50,6 @@ class INFERENCE_ENGINE_API_CLASS(CPUStreamsExecutor) : public IStreamsExecutor {
     int GetStreamId() override;
 
     int GetNumaNodeId() override;
-
-private:
-    struct Impl;
-    std::unique_ptr<Impl> _impl;
```

Comment:
can we just replace this `Impl` with `ov::threading::CPUStreamsExecutor` to avoid strange inheritance?